editor/opening_hours_ui.hpp

```diff
@@ -0,0 +1,45 @@
+#pragma once
+
+#include "std/set.hpp"
+#include "std/vector.hpp"
+
+#include "3party/opening_hours/opening_hours.hpp"
+
+namespace editor
+{
+namespace ui
+{
+
+struct Time
+{
+  int hours;
+  int minutes;
+};
+
+struct Timespan
+{
+  Time from;
+  Time to;
+};
+
+enum class OpeningHoursTemplates
```

Comment:
What is the reason for all these wrapper structures here? Why not add some setters/constructors directly into osmoh::OpeningHours ?